File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/Sensor.java
package org.firstinspires.ftc.teamcode.robot;

import com.qualcomm.robotcore.hardware.AnalogInput;

public class Sensor {
    public AnalogInput sensorRight;
    public AnalogInput sensorLeft;
    Robot robot;

    public Sensor(Robot robot) {
        this.robot = robot;

        this.sensorRight = robot.linearOpMode.hardwareMap.get(AnalogInput.class, "sensor_right");
        this.sensorLeft = robot.linearOpMode.hardwareMap.get(AnalogInput.class, "left");
    }

}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/Sensor.kt
package org.firstinspires.ftc.teamcode.robot

import com.qualcomm.robotcore.hardware.AnalogInput
import com.qualcomm.robotcore.hardware.DcMotor
import com.qualcomm.robotcore.hardware.DcMotorSimple

class Sensor(var robot: Robot) {
    companion object {
        @JvmField
        var pixel = 0;
    }
    public var sensorRight: AnalogInput = robot.linearOpMode.hardwareMap.get(AnalogInput::class.java, "sensor_right")
    public var sensorLeft: AnalogInput = robot.linearOpMode.hardwareMap.get(AnalogInput::class.java, "sensor_left")
}